CHANGELOG.MD

```diff
@@ -1,8 +1,56 @@
 ﻿# CHANGELOG
 
+## [1.24.0](https://github.com/PowerShell/PSScriptAnalyzer/releases/tag/1.24.0)
+
+### What's Changed
+* Update links in module manifest by @martincostello in https://github.com/PowerShell/PSScriptAnalyzer/pull/2034
```

Comment:
Bulk replace `https://github.com/PowerShell/PSScriptAnalyzer/pull/` with `#`. GitHub auto links them.

```suggestion
* Update links in module manifest by @martincostello in #2034
```

Reply:
GitHub may auto link but then the actual document won't have the link if you're reading it anywhere else.

Reply:
We don't publish the changelog anywhere else.